docker/docker-compose.client.yml

```diff
@@ -18,7 +18,7 @@ services:
       - mongodb
     environment:
       LOG_LEVEL: ${LOG_LEVEL:-debug}
-      ZOKRATES_WORKER_HOST: worker
+      CIRCOM_WORKER_HOST: worker
```

Comment:
The availability client test is failing because of this change. It is expecting a different ENV variable to connect to worker. I am fine pushing this as is, and once we deploy to testnet with correct ENV variable (`CIRCOM_WORKER_HOST`), it will work again